### DISCUSSION TITLE
Explanation of Virtual Network Adapter Mode?

### DISCUSSION THREAD

**Discussion by pig0r** [wiresock/WireSockUI#10] [Q&A]
Is there a succinct explanation of exactly what this does?   Does it only route traffic bound to the virtual adapter created by the wireguard installation through the VPN?

**Comment by pig0r** [comment]
Very attentive dev, 10/10.

good to see this application is already abandoned.

**Comment by wiresock** [comment]
While I appreciate your interest, I must clarify that I'm not the original author of WireSockUI. My contribution involved making certain improvements to enhance its convenience. For a more comprehensive understanding of the WireSock VPN Client and its supported modes, I recommend visiting the official WireSock website at https://www.WireSock.net.
